MultiVolumeImporterPlugin.py

```diff
@@ -842,15 +859,28 @@ def initMultiVolumes(self, files, prescribedTags=None):
         continue
 
       tagValues = sorted(tagValue2FileList.keys())
-      # sort the frames
-      firstFrameSize = len(tagValue2FileList[tagValues[0]])
-      frameInvalid = False
-      for tagValue in tagValues:
-        if len(tagValue2FileList[tagValue]) != firstFrameSize:
-          # number of frames does not match
 
-          frameInvalid = True
-      if frameInvalid == True:
+      # Check if the number of frames are the same in each volume
+      slicesPerFrame = {}  # key: number of slices, value: list of tagValue values that has this many slices
+      for tagValue in tagValues:
+        numberOfSlices = len(tagValue2FileList[tagValue])
+        if numberOfSlices in slicesPerFrame.keys():
+          slicesPerFrame[numberOfSlices].append(tagValue)
+        else:
+          slicesPerFrame[numberOfSlices] = [tagValue]
+
+      if len(slicesPerFrame) > 1:
+        # We only accept series that has the same number of slices per frame.
```

Comment:
has -> have

Reply:
I have always been confused about singular vs plural for "series". It is even more confusing here, since the plugin will attempt to match multiple series or a single series into a single multivolume.

Reply:
Yes, this is a shortcoming of the English language.  While we typically insist on using US English words and spelling in names and documentation, I will sometimes use `serie` as a variable name for the singular and `series` for plural in a loop (I understand "serie" is a valid French word).  In comments you can usually make it clear by context.

Reply:
fixed by rephrasing